benchmark/CMakeLists.txt

```diff
@@ -1,2 +1,6 @@
 add_executable(benchncnn benchncnn.cpp)
-target_link_libraries(benchncnn PRIVATE ncnn)
+if(ANDROID)
```

Comment:
```
if(ANDROID_NDK)
```

some arm-linux toolchain file also defines ANDROID, it will break its build for missing libandroid.so
example https://github.com/Tencent/ncnn/blob/master/toolchains/hisiv600.toolchain.cmake

Reply:
谢谢指点，已修改